src/android/io/sqlc/SQLitePlugin.java

```diff
@@ -11,6 +11,9 @@
 import android.util.Base64;
 import android.util.Log;
 
+import com.outsystems.plugins.oslogger.OSLogger;
```

Comment:
This dependency isn't explicit on plugin.xml

Reply:
done ✅